Makefile

```diff
@@ -80,6 +80,26 @@ lint-fix: ## Auto-format all source code + run golangci lint fixers
 build:
 	$(TEMPDIR)/goreleaser build --clean --snapshot
 
+.PHONE: debug
+debug: debug-build debug-run
+
+.PHONY: debug-build
+debug-build:
+	docker build -f Dockerfile-dev -t anchore/harbor-scanner-adapter:debug .
+
+.PHONY: debug-stop
+debug-stop:
+	kubectl delete $(shell kubectl get pods -o name | grep harbor-scanner-anchore)
```

Comment:
is it possible to use a label selector instead of relying on grep?

Reply:
yeah good idea